parsers/mcmaster-parser-2.js

```diff
@@ -19,6 +19,50 @@ function convertTo24Hour(time) {
 }
```

Comment:
always have a space after a comma, this occurs multiple times in the file, fix all